tests/test_sqlalchemy_athena.py

```diff
@@ -584,3 +584,121 @@ def test_create_table_location(self):
                 """
             ),
         )
+
+    def test_create_table_with_comments_compilation(self):
+        dialect = AthenaDialect()
+        table_name = "table_name_000"
+        column_name = "c"
+        table = Table(
+            table_name,
+            MetaData(),
+            Column(column_name, String(10), comment="some descriptive comment"),
+            schema=SCHEMA,
+            awsathena_location="s3://path/to/test_schema/test_create_table/",
+            comment=textwrap.dedent(
+                """
+                Some table comment
+
+                a multiline one that should stay as is.
+                """
+            ),
+        )
+        actual = CreateTable(table).compile(dialect=dialect)
+        self.assertEqual(
+            str(actual),
+            textwrap.dedent(
+                f"""
+                CREATE EXTERNAL TABLE {SCHEMA}.{table_name} (
+                \tc VARCHAR(10) COMMENT 'some descriptive comment'
+                )
+                COMMENT '
+                Some table comment
+
+                a multiline one that should stay as is.
+                '
+                STORED AS PARQUET
+                LOCATION 's3://path/to/test_schema/test_create_table/'\n\n
+                """
+            ),
+        )
+
+    @with_engine()
+    def test_create_table_with_comments(self, engine, conn):
+        table_name = "table_name_000"
+        column_name = "c"
+        comment = textwrap.dedent(
+            """
+            Some table comment
+
+            a multiline one that should stay as is.
+            """
+        )
+        table = Table(
+            table_name,
+            MetaData(),
+            Column(column_name, String(10), comment="some descriptive comment"),
+            schema=SCHEMA,
+            awsathena_location=f"{ENV.s3_staging_dir}/{SCHEMA}/{table_name}",
+            comment=comment,
+        )
+        table.create(bind=conn)
+        check_table = Table(
+            table_name, MetaData(schema=SCHEMA), autoload=True, autoload_with=conn
+        )
+        self.assertIsNot(check_table, table)
+        self.assertIsNot(check_table.metadata, table.metadata)
+        self.assertEqual(
+            check_table.c[column_name].comment, table.c[column_name].comment
+        )
+        # Bug in the API: does not return the exact comment, some blanks are
+        # squashed.
+        # According to cloud trail Athena receives the squashed comment in the
+        # CREATE TABLE request. Could be a bug somewhere in boto.
+        # self.assertEqual(check_table.comment, table.comment)
```

Comment:
Have you confirmed with AWS that this is a bug? You should not write such an uncertain comment.
It should look something like this:
```suggestion
        # The AWS API seems to return comments with squashed whitespace and line breaks.
        # self.assertEqual(check_table.comment, table.comment)
```